### ISSUE TITLE
Datatable data not posted to the server if using it together with Relation Manager

### ISSUE THREAD

**Issue by MaTToX3** [octobercms/october#4776]
OctoberCMS Build: 459
PHP Version: 7.2
Database Engine: MySQL
Plugins Installed:
** Ompmega.MixHelper-1.0.2
** AnandPatel.WysiwygEditors-1.2.9
** RainLab.Translate-1.6.2
** RainLab.Builder-1.0.26
** RainLab.Pages-1.3.1

Related: https://octobercms.com/forum/post/datatable-saving-empty-json-value-on-relation-table

### Description:
When using `datatable` widget together with Relation Manager, the data from widget is not being saved - to be precise, it does not even POST to the server. The same code without Relation Manager works as expected. All the other fields also save as expected.

### Steps to reproduce:
1. Define tab in parent entity in fields.htm:
```
tabs:
    fields:
        fields:
            tab: Fields
            type: partial
            path: $/novelus/forms/controllers/forms/_relation_fields.htm
```

2. Define partial:
`<?= $this->relationRender('fields') ?>`

3. Define fields in child entitiy (only showing the datatable part):
```
select_data:
            label: Data
            span: left
            type: datatable
            tab: Basic
            comment: 'Enter the key-value pairs for available options in select'
            columns:
                value:
                    type: string
                    title: Value
                label:
                    type: string
                    title: Label
            trigger:
                action: show
                field: type
                condition: 'value[select]'
```
4. Add property to `$jsonable` array in model class
`public $jsonable = ['select_data'];`

5. Add field to database table
```
public function up()
{
    Schema::table('novelus_forms_fields', function($table)
    {
        $table->string('select_data');
    });
}
```

6. Try to save 
No error is being shown, no data for this field is being posted to the server:
![image](https://user-images.githubusercontent.com/4114411/69149514-f4872c80-0ad6-11ea-96d8-5d117770485a.png)

**Comment by MaTToX3** [comment]
Can this be confirmed if it is a bug or did I miss something? I am close to the production for this project.

**Comment by MaTToX3** [comment]
Any further info? :)

**Comment by LukeTowers** [comment]
@MaTToX3 if you need priority support, consider signing up for our paid support plan: https://octobercms.com/premium-support. I appreciate the detailed report but I personally don't have time to look into this deeper right now, I'll see if @bennothommo has some time to take a quick look though in the next few weeks.

**Comment by bennothommo** [comment]
@MaTToX3 this is on my to-do list. I will hopefully have a look at it in the next couple of days.

**Comment by bennothommo** [comment]
@MaTToX3 can you please try the fix in #4967 and let us know if that fixes the issue for you?

**Comment by bennothommo** [comment]
Fixed by #4967.
